src/view.js

```diff
@@ -22,6 +22,8 @@ Marionette.View = Marionette.AbstractView.extend({
     this._firstRender = true;
     this._initializeRegions(options);
 
+    this.regions = {};
```

Comment:
The property needs to be initialised to prevent errors when calling `removeRegion`when no regions property exists yet.